### DISCUSSION TITLE
plotting selected differentially abundant taxa

### DISCUSSION THREAD

**Discussion by kusandeep** [FrederickHuangLin/ANCOMBC#38] [Q&A] [answered]
Hi, 
A very beginner question. wondering what's a best way to plot selected differentially abundant taxa/genus using fold change while indicating p-values.. 
  
Thanks and keep up the good work.
Sandeep

**Comment by FrederickHuangLin** [comment, marked as answer]
A bar plot with p-values would be a good choice! For details, you can check this [article](http://www.sthda.com/english/articles/24-ggpubr-publication-ready-plots/76-add-p-values-and-significance-levels-to-ggplots/)

Best,
Huang
